src/components/Navegation.vue

```diff
@@ -30,7 +30,7 @@
             <router-link to="/about" :title="t('button.about')" class="mt-2 md:mt-0 md:mx-4 text-center hover:text-gray-800 dark:hover:text-gray-200">
               {{ t('button.about') }}
             </router-link>
-            <select id="" name="province">
+            <select id="" name="province" style="background: inherit;">
```

Comment:
We are using Tailwinds as the main utility library. To avoid using inline styles, let's replace with the ```bg-transparent``` class to fix the behavior. Let me take the opportunity to explain the main purpose of this field. The curfew's different in some provinces of the country; I would like to allow the users to check the Curfew per province. I'm planning to use a special component for this purpose, not just a Select. I will write an RFC about this proposal.

Reply:
thanks, I'll push the changes later today!